src/main/java/com/study/battleq/modules/board/controller/BoardApiController.java

```diff
@@ -0,0 +1,56 @@
+package com.study.battleq.modules.board.controller;
+
+import com.study.battleq.infrastructure.common.dto.ResponseDto;
+import com.study.battleq.modules.board.controller.request.DeleteSearchRequest;
+import com.study.battleq.modules.board.controller.request.UpdateBoardRequest;
+import com.study.battleq.modules.board.service.dto.response.BoardSearchResponse;
+import com.study.battleq.modules.board.controller.request.CreateBoardRequest;
+import com.study.battleq.modules.board.service.BoardService;
+import com.study.battleq.modules.board.domain.entity.BoardEntity;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+public class BoardApiController {
+
+    private final BoardService boardService;
+
+    @PostMapping("/api/v1/boards")
+    public ResponseDto<Long> saveBoard(@RequestBody @Valid CreateBoardRequest request) {
+        Long id = boardService.save(request.toDto());
+        return ResponseDto.ok(id);
+    }
+
+    @GetMapping("/api/v1/boards")
+    public ResponseDto<List> findAllBoard(){
+        List<BoardSearchResponse> boardSearchResponseList = boardService.findAll();
+        return ResponseDto.ok(boardSearchResponseList);
```

Comment:
Inline으로 한번에 가도 깔끔할듯!
```suggestion
        return ResponseDto.ok(boardService.findAll());
```
추 후 페이징이 필요할테니 메소드 파라미터에 Pageable을 넣을지 int page, int size 이런식으로 넣을지 고민해보시는 것도 좋을 것 같습니다. 스웨거에 Pageable 넣으면 깨지는 경우가 있어서 방법이 있긴 한거 같은데 저는 아직 못찾았네용 ㅎ ㅠ

Reply:
페이징 관련 
배틀큐 특징 베이스로 프론트와 협의하여 진행하겠습니다.